models/gas/arbitrum/gas_arbitrum_fees_traces.sql

```diff
@@ -13,55 +14,57 @@ WITH traces AS (
      SELECT traces.block_time
      , traces.block_number
      , traces.tx_hash
-     , MAX(traces.from) AS trace_from
+     , MAX(traces."from") AS trace_from
      , MAX(traces.to) AS trace_to
      , traces.trace
      , MAX(traces.input) AS trace_input
-     , substring(MAX(traces.input),1,10) AS trace_method
+     , bytearray_substring(MAX(traces.input),1,4) AS trace_method
      , SUM(traces.gas_used_original) AS gas_used_original
      , SUM(traces.gas_used_trace) AS gas_used_trace
      , MAX(traces.trace_type) AS trace_type
      , MAX(traces.trace_value) AS trace_value
      , MAX(traces.trace_success) AS trace_success
      , MAX(traces.tx_success) AS tx_success
      FROM (
-          SELECT from
+          SELECT "from"
           , to
           , tx_hash
           , trace_address AS trace
-          , gas_used AS gas_used_original
-          , gas_used AS gas_used_trace
+          , cast(gas_used as double) AS gas_used_original
+          , cast(gas_used as double) AS gas_used_trace
           , block_time
           , block_number
           , input
           , type AS trace_type
+          , call_type
```

Comment:
Our migration validation checks won't like schema changes (both type changes and new/removed columns).
It's probably preferred to keep all the migration changes in 1 PR do any modifications/additions in a follow up PR.

Reply:
cool i'll remove for now